service/provider/src/main/java/gov/va/vro/service/provider/mas/service/MasApiService.java

```diff
@@ -87,18 +89,33 @@ public String orderExam(MasOrderExamReq masOrderExamReq) throws MasException {
     try {
       String url = masApiProps.getBaseURL() + masApiProps.getCreateExamOrderPath();
       HttpHeaders headers = getMasHttpHeaders();
+      ObjectMapper mapper = new ObjectMapper();
+
+      try {
+        // convert user object to json string and return it
+        log.info("masOrderExamReq JSON : " + mapper.writeValueAsString(masOrderExamReq));
+      } catch (JsonGenerationException | JsonMappingException e) {
+        // catch various errors
+        // NOP;
+      }
+      log.info(" Exam Order >>>> API Service URL : " + url.toString());
+      log.info(
+          " Exam Order >>>> API Service collectionsid : "
+              + masOrderExamReq.getCollectionsId().toString());
+      log.info(
+          " Exam Order >>>> API Service condition text: "
+              + masOrderExamReq.getConditions().get(0).getContentionText());
```

Comment:
We should guard against the case where the contention list is empty.
Also, are we sure the contention text does not contain PII?